README.md

```diff
@@ -1,6 +1,53 @@
-# FINDOLOGIC export toolkit
+FINDOLOGIC export toolkit
+=========================
 
 [![Travis](https://img.shields.io/travis/findologic/libflexport.svg)](https://travis-ci.org/findologic/libflexport)
 [![Latest Stable Version](https://poser.pugx.org/findologic/libflexport/v/stable)](https://packagist.org/packages/findologic/libflexport)
 
-**Highly experimental! Don't use it!**
\ No newline at end of file
+Synopsis
+========
+
+This project provides a export library for XML and CSV generation according to the FINDOLOGIC export patterns.
+* XML: <https://docs.findologic.com/doku.php?id=export_patterns:xml>
+* CSV: <https://docs.findologic.com/doku.php?id=export_patterns:csv>
+
+
+Basic usage
+============
+
+```PHP
```

Comment:
Writing the name of the language in lower case is more common. Not sure though, if GitHub is case sensitive in that regard.

Reply:
Done